src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_params.py

```diff
@@ -180,3 +180,19 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
     with self.argument_context('iot hub manual-failover') as c:
         c.argument('failover_region', options_list=['--failover-region', '--fr'], help='The region that the IoT hub'
                    'fails over to. Must be the paired region to the current IoT hub region.')
+
+    # Arguments for IoT Digital Twin
+    with self.argument_context('iot pnp') as c:
+        c.argument('repo_endpoint', options_list=['--repo-rp', '--rp', '--endpoint', '-e'], help='PnP endpoint.')
```

Comment:
why multiple long options? 

something like the following is simpler:
```suggestion
        c.argument('repo_endpoint', options_list=['--endpoint', '-e'], help='PnP endpoint.')
```

Also should pnp be clarified?

Reply:
Changed